chia/cmds/configure.py

```diff
@@ -71,13 +72,37 @@ def configure(
         else:
             print("uPnP disabled")
         change_made = True
+    if testnet is not None:
+        testnet_port = "58444"
+        testnet_introducer = "beta1_introducer.chia.net"
+        testnet = "testnet7"
+        config["full_node"]["port"] = int(testnet_port)
+        config["full_node"]["introducer_peer"]["port"] = int(testnet_port)
+        config["farmer"]["full_node_peer"]["port"] = int(testnet_port)
+        config["timelord"]["full_node_peer"]["port"] = int(testnet_port)
+        config["wallet"]["full_node_peer"]["port"] = int(testnet_port)
+        config["wallet"]["introducer_peer"]["port"] = int(testnet_port)
+        config["introducer"]["port"] = int("58444")
```

Comment:
to be consistent with the rest?
```suggestion
        config["introducer"]["port"] = int(testnet_port)
```